code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -14,8 +14,8 @@
 	if(iscrowbar(W))
 		new /obj/item/stack/sheet/wood(src)
 		var/turf/T = get_turf(src)
-		for(var/obj/O in contents)
-			O.forceMove(T)
+		for(var/atom/movable/AM in contents)
+			AM.forceMove(T)
```

Comment:
Many effects, including the fire overlay and firelightdummy, may fall into this bucket. Test crowbarring one of these crates open while it is on fire or has another effect/overlay present to make sure it doesn't result in any issues.

Reply:
fire stuff doesn't appear outside the crate when crowbar'd open at all, not even in a vv of turf contents